### PR TITLE
New search-input component

### PR DESCRIPTION
**Pull request by dunnkers**
New component used by resource-manager.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-railio-form-components",
-  "version": "0.0.11",
+  "version": "0.0.12",
```
Comment (cbroeren) [inline package.json]: Next time. you don't need to update the version or add a tag on every project. It will be done after merging :-)

**Comment by dunnkers** [comment]
@cbroeren is it possible that version 0.0.8 of ember-railio-formatting (f374e2dc2d393baad1acebbfe74835ceba6e8568) caused a test to fail?

Travis is not happy.
https://travis-ci.org/Fabriquartz/ember-railio-form-components/jobs/83300065 

```
not ok 78 PhantomJS 1.9 - Integration | Component | {{date-picker}}: typing in date shorthands
    ---
        actual: >
            05-10-15
        expected: >
            05-010-15
        Log: |
    ...
```

:smile_cat:

**Comment by cbroeren** [comment]
@dunnkers, I just fixed this, the test was not OK for 2 digit months (like oktober ;-))
You could pull the new commits and rebase with master

**Comment by dunnkers** [comment]
Nice :smile:  Pull request should be fine now.

**Comment by martndemus** [comment]
Can you remove the merged commits, and rebase instead?

**Comment by martndemus** [comment]
Merged via #10

**Pull request closed without merging**